lib/src/defaults/providers/rest/src/rest_external_interface.dart

```diff
@@ -33,10 +35,42 @@ class RestExternalInterface
         send(RestSuccessResponse(data: data));
       },
     );
+    on<BinaryDataRestRequest>(
+      (request, send) async {
+        final binaryData = File(request.src).readAsBytesSync();
```

Comment:
You can refer to this PR, for conditionally importing File.
https://github.com/dnfield/flutter_svg/pull/613